2020-InfiniteRecharge/src/main/java/frc/robot/subsystems/Chassis.java

```diff
@@ -154,10 +162,15 @@ public void DriveSystem(Joystick drive)
 
     public void DriveSystem(Joystick drive, String mode)
     {   
+        // graphing the motor parameters
         motor1Temp = sparkMax1.getMotorTemperature();
         motor2Temp = sparkMax2.getMotorTemperature();
         motor3Temp = sparkMax3.getMotorTemperature();
         motor4Temp = sparkMax4.getMotorTemperature();
+        motor1Temp = sparkMax1.getOutputCurrent();
+        motor2Temp = sparkMax2.getOutputCurrent();
+        motor3Temp = sparkMax3.getOutputCurrent();
+        motor4Temp = sparkMax4.getOutputCurrent();
```

Comment:
Be sure to set `motor1current` rather than `motor1Temp` here. Otherwise, this looks good to go once it is tested.

Reply:
Fixed the variables